loopy/target/c/codegen/expression.py

```diff
@@ -685,7 +685,7 @@ def map_min(self, expr, enclosing_prec):
 
     def map_if(self, expr, enclosing_prec):
         from pymbolic.mapper.stringifier import PREC_NONE
-        return "({} ? {} : {})".format(
+        return "(({}) ? {} : {})".format(
                 self.rec(expr.condition, PREC_NONE),
```

Comment:
You can force parentheses more cleanly (IMO) by passing a different (higher) precedence.

Reply:
Would that be the highest precedence (PREC_CALL)?

Reply:
Done in 22e1119a